modules/activitypub/comment.go

```
@@ -0,0 +1,41 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package activitypub
+
+import (
+	"context"
+	"strconv"
+	"strings"
+
+	"code.gitea.io/gitea/models/issues"
+	repo_model "code.gitea.io/gitea/models/repo"
+	"code.gitea.io/gitea/modules/log"
+
+	ap "github.com/go-ap/activitypub"
+)
+
+// Create a comment
+func Comment(ctx context.Context, note ap.Note) {
+	actorUser, err := personIRIToUser(ctx, note.AttributedTo.GetLink())
+	if err != nil {
+		log.Warn("Couldn't find actor", err)
```

I think you want to return a error for these functions. As there are other possible errors in this code. 
Don't forget to add `return` for this line.

---

Ditto this for the other new functions.